rero_ils/modules/acq_order_lines/jsonschemas/acq_order_lines/acq_order_line-v0.0.1.json

```diff
@@ -150,6 +145,18 @@
         }
       }
     },
+    "document": {
```

Comment:
description is missing. This is the case for almost all the field in this schema. I would be great to have it.

Reply:
yes, the description field is missing in many resources schema.
Maybe we can ask the missing description to the PO group?